index.js

```diff
@@ -8,7 +8,10 @@ var CacheStore = require('./cache_store');
 var DEFAULT_TOTAL_VNODES = 1024;
 var DEFAULT_CALM_THRESHOLD = 500;
 var DEFAULT_RETRY_INTERVAL_MS = 5000;
-var MAX_PARALLEL_TASKS = 10;
+
+// We're setting a modest limit to make sure we're not starving the event-loop of CPU
+// with overly aggressive setting
+var MAX_PARALLEL_TASKS = 2;
```

Comment:
Because making changes to the Sevnup library is expensive (in terms of process), can we instead define this as an instance field whose value is passed in by Demand (Demand -> SevnupNamespace -> Sevnup), the same way for example `this.releaseKeyCallback = params.releaseKey;` is passed.

Then in Demand we can have this be flipr configurable so that if we need to ramp up or ramp it down its in our control

Reply:
+1, it would be nice if we can change this number dynamically at runtime.

Reply:
Good call, not sure if it's worth the hustle to pull it through Flipr, but def can make at a configurable through creation params

Reply:
Even though we don't really want to have this configurable, as this is fairly intricate parameter